pkgs/by-name/vf/vfkit/support-apple-11-sdk.patch

```diff
@@ -0,0 +1,827 @@
+diff --git vendor/github.com/Code-Hex/vz/v3/virtualization_12.m vendor/github.com/Code-Hex/vz/v3/virtualization_12.m
+index 4bcbcfb..8cecc78 100644
+--- vendor/github.com/Code-Hex/vz/v3/virtualization_12.m
++++ vendor/github.com/Code-Hex/vz/v3/virtualization_12.m
+@@ -8,6 +8,7 @@
+ 
+ bool vmCanStop(void *machine, void *queue)
+ {
++#ifdef INCLUDE_TARGET_OSX_12
```

Comment:
This could be filed upstream at `github.com/Code-Hex/vz` (no idea if this would be accepted or not though )